server/task_check_executor_statement_advisor.go

```diff
@@ -59,7 +63,56 @@ func (exec *TaskCheckStatementAdvisorExecutor) Run(ctx context.Context, server *
 		return []api.TaskCheckResult{}, common.Errorf(common.Internal, fmt.Errorf("failed to check statement: %w", err))
 	}
 
+	return generateResultList(adviceList), nil
+}
+
+// RunSchemaReview will run the schema review check according to schema review policy.
+func (exec *TaskCheckStatementAdvisorExecutor) RunSchemaReview(ctx context.Context, server *Server, taskCheckRun *api.TaskCheckRun) (result []api.TaskCheckResult, err error) {
+	payload := &api.TaskCheckDatabaseStatementAdvisePayload{}
+	if err := json.Unmarshal([]byte(taskCheckRun.Payload), payload); err != nil {
+		return []api.TaskCheckResult{}, common.Errorf(common.Invalid, fmt.Errorf("invalid check statement advise payload: %w", err))
+	}
+
+	if payload.DbType != db.MySQL && payload.DbType != db.TiDB {
+		return []api.TaskCheckResult{}, common.Errorf(common.NotImplemented, fmt.Errorf("not implemented schema review for %s yet", payload.DbType))
+	}
```

Comment:
Seems we shall put the DB check inside each rule, as the schema review rule is related to the DB type. And we also need a mapping about `SchemaReviewRuleType` - `db.Type`

Reply:
Got it